operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+
+## Governance Remediation Process
+
+### Roles
+
+* TAG Contributor Strategy: For missing governance the assignee will lead, consulting and informing with the assigned TOC member as needed. For non-practiced governance, they act as an advisor to the TOC
+* TOC member assignee(s): serves an oversight/informational function that the governance assignee can leverage as needed to impress upon the project the needed changes or leads the discussion with the project in the case of non-practiced governance
+* Code of Conduct Committee: if a failure of governance coincides with CoC violations by one or more project leaders, the CoCC will coordinate with the assessment team.
+
+### Missing Governance
+
+1. Assessment Team Formation: The TOC will assign one TAG Contributor Strategy assignee and one TOC member.
+
+2. Investigation: The assessment team will Interview the reporter, project maintainers, and other relevant contributors to determine if the governance issue is valid and assess the project's ability to resolve it internally.
+
+3. Recommendation Development: In collaboration with the project, the assessment team will draft new governance policies to address gaps. This will include an actionable plan for the project to follow, with specific milestones and deadlines.
+
+4. Implementation and Monitoring: The assessment team will regularly check in on the project's progress towards defined milestones, provide additional support as needed, and provide a public progress report at 3 and 6 month milestones.
+
+5. Resolution or Escalation: If the project meets the milestones, the issue can be resolved. If not, it should be transferred to the Non-Practiced Governance process.
+
+### Governance Remediation Process: Non-Practiced Governance
+
+1. Triage: The TOC will do an initial triage of the issue and assign a lead member and a supporting member to guide the process.
+
+2. Assessment Team Formation: The two TOC members will work with a member of [Governance WG](https://contribute.cncf.io/about/governance/) to form an assessment team.
+
+3. Investigation: The assessment team will Interview the reporter, project maintainers, and other relevant contributors to determine if the governance is not being practiced and assess the project's ability to resolve it internally. This may include attendance by the project’s leadership team to a private TOC meeting to discuss the issue more in depth.
+
+4. Recommendation Development: The assessment team will determine where the project is not following its documented governance and create recommendations on how to get back on track. This will include an actionable plan for the project to follow, with specific milestones and deadlines. These recommendations will then be shared with the whole TOC for further feedback and commentary. The TOC will make a final decision and provide directives to the project.
```

Comment:
```suggestion
4. Recommendation Development: The assessment team will determine where the project is not following its documented governance and work with the project to create recommendations on how to get back on track. This will include an actionable plan for the project to follow, with specific milestones and deadlines. These recommendations will then be shared with the whole TOC for further feedback and commentary.
```